### PR TITLE
Insert human curated 1.11 release notes and set current version

### PR DESCRIPTION
**Pull request by calebamiles**
**What this PR does / why we need it**:

Inserts human curated release notes

**Special notes for your reviewer**:

None

**Release note**:

NONE

### CODE REVIEW

File: CHANGELOG-1.11.md [hunk 1]
```diff
@@ -114,21 +154,663 @@ filename | sha256 hash
...
+* **[Pod priority and preemption](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/) is now enabled by default. Note that this means that pods from *any* namespace can now request priority classes that compete with and/or cause preemption of critical system pods that are already running. If that is not desired, disable the PodPriority feature by setting `--feature-gates=PodPriority=false` on the kube-apiserver, kube-scheduler, and kubelet components before upgrading to 1.11. Disabling the PodPriority feature limits [critical pods](https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/#marking-pod-as-critical-when-priorites-are-enabled) to the `kube-system` namespace.
```
Comment (liggitt) [inline CHANGELOG-1.11.md]: looks like it's missing a closing bold tag

Comment (liggitt) [reply CHANGELOG-1.11.md]: probably add after `that are already running.**`

Comment (jberkus) [reply CHANGELOG-1.11.md]: after "default"

Comment (calebamiles) [reply CHANGELOG-1.11.md]: fixed

**Pull request merged**